tests/toolchain/gcc_fortran_compilation.pm

```diff
@@ -48,6 +48,9 @@ sub run {
 
     save_screenshot;
     script_run 'popd';
+
+    script_run 'free -m';
+    save_screenshot;
```

Comment:
You are adding the same two lines to two modules and in both cases it is not clear from the code why they are there. Would it work to add them to a common post fail hook?
I am fine to also do it just temporarily with a comment referencing the progress ticket but maybe the right task would be to get a ppc64le worker. Others have one available so it should be feasible to achieve and might be a worthwhile effort investment for you :)

Reply:
I want to inspect the RAM not on failure, in normal working. I can add the progress ticket in a comment. 
Sure, I will ask admins of our ppc64le machine in order to have the verification run.